chain/ethereum/src/network_indexer/convert.rs

```diff
@@ -34,6 +34,46 @@ impl ToEntityKey for &BlockWithOmmers {
     }
 }
 
+impl ToEntityId for LoadedTransaction {
```

Comment:
@lutter How would we best deal with the fact that references are strings but the IDs should be represented as `Bytes!`?

Do we keep this `ToEntityId` impl for references but when we create the `Transaction` and `Log` entities and entity keys for them, we generate the corresponding value as `Bytes`?